configuration/override_dir_structure.rst

```diff
@@ -80,6 +82,22 @@ method::
 
 Here you have changed the location of the directory to ``app/{environment}/logs``.
 
+.. _override-templates-dir:
+
+Override the Templates Directory
+--------------------------------
+
+If your templates are not stored in the default ``app/Resources/views/``
+directory, use the :ref:`twig.paths <config-twig-paths>` configuration option to
+define your own templates directory (or directories):
```

Comment:
>What about mentioning the root dir here? Either via a note or %kernel.root_dir%/Resources/views/ instead of app/Resources/views/. 

I agree we should explicit it at some point but I don 't think this is the right place to do so, since other resources are concerned (i.e translations).

The whole `Resources` folder should be relative to the kernel root dir.